torch_tvm/register.cpp

```diff
@@ -72,5 +75,33 @@ PYBIND11_MODULE(_torch_tvm, m) {
 
   m.def("disable", []() { fusion_enabled = false; });
 
+  m.def("push_relay_expr", [](std::shared_ptr<Graph> g,
+                              std::vector<at::Tensor> inputs) {
+    std::cerr << *g << "\n";
+    for (auto node : g->nodes()) {
+      if (node->kind() == tvm_sym) {
+        std::vector<Value*> v;
+        auto subgraph = node->g(attr::Subgraph);
+        AT_CHECK(subgraph->inputs().size() == inputs.size(), "Expected ", subgraph->inputs().size(), " inputs");
+        for (auto i = 0; i < inputs.size(); ++i) {
+          subgraph->inputs()[i]->inferTypeFrom(inputs[i]);
+        }
+        TVMContext ctx;
+        ctx.device_type = kDLCPU;
+        ctx.device_id = 0;
+        auto expr = TVMCompiler::convertToRelay(subgraph, ctx);
+        relay_exprs[++relay_exprs_uuid] = expr;
+        return relay_exprs_uuid;
```

Comment:
that will only convert the first subgraph to relay right? Shall we give the user warning here indicating they could not convert the full graph to relay, they will need to add more operators to make sure a single subgraph